### ISSUE TITLE
Ensure all host prerequisites are installed from one playbook

### ISSUE THREAD

**Issue by spisarski** [cablelabs/snaps-boot#99]
The playbooks that are responsible for setting up the kolla node hosts do many of the exact same things which are primarily apt and pip installations. The affected files should all be in the following directory: snaps_openstack/ansible_p/commission/openstack/playbooks/deploy_mode/kolla

**Comment by spisarski** [comment]
Once this has been completed, this operation should be performed in parallel on all nodes being installed which should help speed things up.

**Comment by issi7915** [comment]
Should issue 41 (#105) remain in progress or is this a separate issue?

**Comment by spisarski** [comment]
@issi7915, thanks for checking the issues, but this one has to do with prerequisites for the nodes being deployed which is currently being done sequentially rather than in parallel. #41/105 are for installing the prerequisites on the build server.

**Comment by RandyLevensalor** [comment]
@Sanjhana-Jayagopal now that the preRequsite.sh is down to apt installing 3 packages, these can be moved to an Ansible playbook.

This playbook should be called when the -p option is called.  

I don't think that we should remove these packages with the -pc option, since it is possible that these packages were installed without using snaps-boot.

**Comment by RandyLevensalor** [comment]
Add the python to call the playbook to: snaps_boot/ansible_p/commission/install
Add the playbook to : snaps_boot/ansible_p/commission/install/playbooks

**Comment by bo-quan** [comment]
Shifting to digital rebar, so close it for now.
